doc/release-notes/0.27/0.27.md

```diff
@@ -36,7 +36,7 @@ To learn more about support for OpenJ9 releases, including OpenJDK levels and pl
 
 ## Notable changes in this release
 
-The following table covers notable changes in v0.27.0. Further information about these changes can be found in the [user documentation](https://eclipse-openj9.github.io/openj9-docs/version0.27/).
+The following table covers notable changes in v0.27.0. Further information about these changes can be found in the [user documentation](https://www.eclipse.org/openj9-docs/version0.27/).
```

Comment:
The new URL should be https://www.eclipse.org/openj9/docs/version0.27/.

Reply:
Can you also please add the missing `</tbody>` for the first table.

Reply:
Updated.